internal/indexmanagement/reconcile.go

```diff
@@ -366,21 +366,16 @@ func (imr *IndexManagementRequest) reconcileIndexManagementCronjob(policy apis.I
 
 func formatCmd(policy apis.IndexManagementPolicySpec) string {
 	cmd := []string{}
-	result := []string{}
 	if policy.Phases.Delete != nil {
-		cmd = append(cmd, "./delete", "delete_rc=$?")
-		result = append(result, "exit $delete_rc")
+		cmd = append(cmd, "./delete")
 	}
 	if policy.Phases.Hot != nil {
-		cmd = append(cmd, "./rollover", "rollover_rc=$?")
-		result = append(result, "exit $rollover_rc")
+		cmd = append(cmd, "./rollover")
 	}
 	if len(cmd) == 0 {
 		return ""
 	}
-	cmd = append(cmd, fmt.Sprintf("$(%s)", strings.Join(result, "&&")))
-	script := strings.Join(cmd, ";")
-	return script
+	return strings.Join(cmd, " && ")
```

Comment:
sorry, i don't think i was clear enough before... this is not what we want, if delete fails we still want to try to rollover and using `&&` like this will prevent it.

I'm suggesting that we instead have a new command... and that command that is called is (currently) a bash script that will call delete and then call rollover and this command's RC will be based on $? from both the delete and rollover function call.